### DISCUSSION TITLE
Example Change #3: Provider Activity ID description typo

### DISCUSSION THREAD

**Discussion by robredpath** [IATI/standard-process#6] [Demos and examples]
In our meeting earlier this month, we looked at a very small issue that our current process doesn't support very well. This is an example of the kind of real-world issues that we deal with in standard governance.

We've copied the example into this repo as an example change request: https://github.com/IATI/standard-process/blob/main/example_changes/provider_acitivity_id_desc.md

It would be useful to hear views from the group on what IATI's response to this issue should be. We can set aside for a moment the existing upgrade process and consider what we think a good response looks like. As we discuss, we may uncover principles that can contribute to the discussion about principles that @stevieflow has opened.

**Comment by robredpath** [comment]
In the meeting, it was suggested that a suitable response was that this could be changed by the Secretariat without any additional authorisation, as long as appropriate notice was given of the forthcoming change to allow any objections to be raised. A period of 45 days was suggested for this. 

It would useful to hear any further reflections on this.

**Reply by bill-anderson** [reply]
Agree 100%

**Comment by davidmegginson** [comment]
Ditto -- I think this is a clear case where we don't want to crush a simple and obvious fix under the weight of excessive process.
